tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -1569,20 +1569,6 @@ def test_keras_autolog_infers_model_signature_correctly_with_keras_sequence(
         )
 
 
-def test_keras_autolog_does_not_log_model_signature_when_mlflow_autolog_called(
-    keras_data_gen_sequence,
-):
-    mlflow.autolog()
-    initial_model = create_tf_keras_model()
-    initial_model.fit(keras_data_gen_sequence)
-
-    mlmodel_path = mlflow.artifacts.download_artifacts(
-        f"runs:/{mlflow.last_active_run().info.run_id}/model/MLmodel"
-    )
-    mlmodel_contents = yaml.safe_load(open(mlmodel_path, "r"))
-    assert "signature" not in mlmodel_contents, mlmodel_contents.keys()
```

Comment:
Can we convert this to a `test_keras_autolog_logs_model_signature` test case?

Reply:
added with signature check and signature content validation